docs/canvas/canvas-workpad.asciidoc

```diff
@@ -1,49 +1,68 @@
 [role="xpack"]
 [[canvas-workpad]]
-=== Using the workpad
+=== Create a workpad
 
-Now that you have a workpad with sample data that you can mess with, let’s mess with it. 
-We’ll start out by making a few stylistic changes. 
+When you create a workpad, you'll start with a blank page, or you can choose a workpad that is already started for you. 
 
-. Click the gauge chart in the top left of the workpad (fun fact, these are actually pie charts). 
-Try clicking on the small red slice. If you click the number, you'll notice that
-you’ve actually selected a number element, not the gauge itself. 
-+ 
-This element is now selected. Off to the right, you'll see that the side bar has changed. 
-This is where you can make changes to the element.
-+
-[role="screenshot"]
-image::images/canvas_workpad_edit_style.png[]
+. Open *Canvas*.
 
-. Try changing the color palette. Easy right? Click the Back button in your 
-browser to undo your change.
+. Choose one of the following options:
 
-. Try dragging the chart around the page. Grab the resize handles to make 
-the chart bigger and smaller.
+* To start with a single, blank workpad, click *Create workpad*.
 
-. Now click somewhere off of the element, but not on another element, 
-to deselect it. 
-+
-You will notice the sidebar changing again. You now have 
-access to the page and workpad-level settings. Feel free to change the page 
-background color or rename your workpad.
-
-Speaking of pages, Canvas workpads can have multiple pages.
-
-. Click the arrow to the right of the *Page 1 of 2* text in the toolbar at the bottom
-of the page. You'll see a whole new view of the the eCommerce data.
-. Click the *Page 2 of 2* text to see a live preview of all of the pages in your 
-workpad.
-. Click the (+) button to create a new, blank page.
-. Click the *Add element* button in the top right hand corner of the page.
-. Add an *Area Chart.*
+* To create a workpad from a template, select *Templates*, then select a preconfigured template.
```

Comment:
Do you think providing a  link to preconfigured templates from here would be a good idea ?

Reply:
Since users can access the preconfigured templates from the UI, and they have already been instructed to open Canvas, I think we can leave out the link to the templates. Our docs seem to already be link-heavy, and I don't want to redirect them from this page.